prow/jobs/test-infra/run-terraform-config.yaml

```diff
@@ -21,23 +21,50 @@ presubmits: # runs on PRs
         - ^main$
       spec:
         containers:
-          - image: "eu.gcr.io/kyma-project/test-infra/terraform:PR-6790"
+          - image: "europe-docker.pkg.dev/kyma-project/dev/test-infra/terraform:PR-6893"
             securityContext:
               privileged: false
-            workingDir: "/home/prow/go/src/github.com/kyma-project/test-infra/configs/terraform/secrets-leaks-log-scanner"
             command:
-              - "/tfcmt/tfcmt"
+              - "/bin/sh"
             args:
-              - "-owner"
-              - "$(REPO_OWNER)"
-              - "-repo"
-              - "$(REPO_NAME)"
-              - "-pr"
-              - "$(PULL_NUMBER)"
-              - "plan"
-              - "--"
-              - "terraform"
-              - "plan"
+              - "-c"
+              - "> /bin/terraform -chdir=/home/prow/go/src/github.com/kyma-project/test-infra/configs/terraform/secrets-leaks-log-scanner init && /tfcmt/tfcmt -owner $REPO_OWNER -repo $REPO_NAME -pr $PULL_NUMBER plan -- terraform -chdir=/home/prow/go/src/github.com/kyma-project/test-infra/configs/terraform/secrets-leaks-log-scanner plan -input=false"
```

Comment:
I don't feel like it's a way on how you should handle paths in the terraform. First of all - your current dir already is `/prow/go/src/github.com/kyma-project/test-infra`. According to the docs you can use relative paths. Please switch `-chdir` to `configs/terraform/secrets-leaks-log-scanner`.

Reply:
Why using a full path is a problem?
Terraform documentation says.
> Global options (use these before the subcommand, if any):
  -chdir=DIR    Switch to a different working directory before executing the
                given subcommand.

Reply:
It's to keep things as simple as possible and to prevent job from breaking in case the absolute path changes.